qc_grader/challenges/programming_challenge_aus/problem_set.py

```diff
@@ -42,9 +44,16 @@ def grade_prob2_ex1(tsp_function: Callable) -> None:
 
 
 @typechecked
-def grade_prob2_ex2(result, solution_order: Tuple, solution_distance: float) -> None:
+def grade_prob2_ex2(result: SamplingVQEResult) -> None:
+    
+    tsp = Tsp.create_random_instance(3, seed=123)
+    adj_matrix = nx.to_numpy_matrix(tsp.graph)
```

Comment:
hello @Astri-Cornish if possible can you submit PRs instead of pushing directly. multiple challenges and problem sets rely on this grader client so we want to try to minimize disruptions as best as we can.

it is good to get a second pair of eyes on an update. for example, this is going to cause errors because 
1. there is no import for `nx` 
2. `to_numpy_matrix` has been removed in networkx 3

in addition, updates to grader client to do not get immediately deployed. any changes being committed that we are unaware of may sit undeployed

Reply:
Woops sorry yes I don't know what I was thinking with merging straight away!